docs/recipes/Zustand.md

```diff
@@ -0,0 +1,688 @@
+  } catch (error) {
+    return { textLabel: "", accessibilityLabel: "" };
+  }
+};
+
+export const getDuration = (episode: Episode) => {
+  const seconds = Number(episode.enclosure.duration);
+  const h = Math.floor(seconds / 3600);
+  const m = Math.floor((seconds % 3600) / 60);
+  const s = Math.floor((seconds % 3600) % 60);
+
+  const hDisplay = h > 0 ? `${h}:` : "";
+  const mDisplay = m > 0 ? `${m}:` : "";
+  const sDisplay = s > 0 ? s : "";
+  return {
+    textLabel: hDisplay + mDisplay + sDisplay,
+    accessibilityLabel: translate("demoPodcastListScreen.accessibility.durationLabel", {
+      hours: h,
+      minutes: m,
+      seconds: s,
+    }),
+  };
+};
+```
+
+</details>
+
+## Remove Mobx-State-Tree
+
+Now that our models have been converted, follow our recipe to [Remove Mobx-State-Tree](./RemoveMobxStateTree.md) entirely from your project.
+
+## Create Store
+
+Let's create our main Zustand store. Create a new file `app/store/RootStore.ts`:
+
+**`app/store/RootStore.ts`**
+
+```ts
+import { create } from "zustand";
+import { useShallow } from "zustand/react/shallow";
+import { AuthenticationStore, authenticationStoreSelector, createAuthenticationSlice } from "./AuthenticationStore";
+import { EpisodeStore, createEpisodeSlice, episodeStoreSelector } from "./EpisodeStore";
+
+export interface RootStore extends AuthenticationStore, EpisodeStore {}
+
+export const useStore = create<RootStore>()((...a) => ({
+  ...createAuthenticationSlice(...a),
+  ...createEpisodeSlice(...a),
+  // add your state slices here
+}));
+
+// optional: custom hooks can be used to pick pieces from state
+// useShallow is used to help prevent unnecessary rerenders
+export const useAuthenticationStore = () => useStore(useShallow(authenticationStoreSelector));
+export const useEpisodeStore = () => useStore(useShallow(episodeStoreSelector));
+```
+
+- We're combining AuthenticationStore and EpisodeStore into one Zustand store for simplicity. Again, Zustand is very non-opinionated so you can modify this structure if desired.
+- `useAuthenticationStore` and `useEpisodeStore` are exported as custom hooks to make it easier to select common pieces of state. Feel free to create additional custom hooks for reusable lookup patterns and prevent unnecessary re-renders [Read more about this](https://github.com/pmndrs/zustand?tab=readme-ov-file#selecting-multiple-state-slices).
+
+Create `store/index.ts` file to export our hooks and selectors for easy use across our app:
+
+**`app/store/index.ts`**
+
+```ts
+export * from "./RootStore";
+export * from "./AuthenticationStore";
+export * from "./EpisodeStore";
+export * from "./Episode";
+```
+
+## Use Zustand in Components
+
+Zustand's hooks-based API makes it easy to pull data into components.
+
+In the Ignite Demo App, we'll update the following components to use our exported Zustand hooks and selectors:
+
+**`app/navigators/AppNavigator.tsx`**
+
+```tsx
+import { useStore, isAuthenticatedSelector } from "../store";
```

Comment:
There is a mix of TS alias paths and relative paths, you can probably stick with the alias `app/store` as it probably helps reduce maintenance updates (or if people move things around)